### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by saimaghafoor**
Updated roadmap including:

- dates
- things we're working on
- things we've done

### CODE REVIEW

File: app/templates/views/guidance/features/roadmap.html [hunk 1]
```diff
@@ -29,29 +29,27 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Let services create letter templates that comply with the <a class="govuk-link govuk-link--no-visited-state" href="https://law.gov.wales/culture/welsh-language/welsh-language-act-1993">Welsh Language Act 1993</a></li>
+    <li>Make it easier for users to joing a live service</li>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/features/roadmap.html:32]:
```suggestion
    <li>Make it easier for users to join a live service</li>
```

File: app/templates/views/guidance/features/roadmap.html [hunk 2]
```diff
@@ -15,8 +15,8 @@ <h1 class="heading-large">Roadmap</h1>
 
   {{ content_metadata(
     data={
-      "Last updated": "8 February 2023",
-      "Next review due": "26 April 2023"
+      "Last updated": "2 May 2023",
```
[the remark this answers is not in the document]
Comment (saimaghafoor) [reply app/templates/views/guidance/features/roadmap.html:18]: Yes! I wanted to get it to you so you had enough time to review and for any iterations ready for 2nd May

File: app/templates/views/guidance/features/roadmap.html [hunk 3]
```diff
@@ -60,13 +58,8 @@ <h2 class="heading-medium" id="things-we-have-done">Things we’ve done</h2>
...
 
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2023/01/23/how-we-let-gov-uk-notify-users-set-their-own-branding/">upload their own logo for emails and letters</a></li>
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for("main.guidance_receive_text_messages") }}">receive text messages</a> without having to contact the Notify team</li>
-    <li>Updated the information and guidance on the Notify website</li>
-    <li>Made it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for("main.guidance_trial_mode") }}">trial mode restrictions</a></li>
-    <li>Designed a first draft of reusable content patterns for text messages</li>
-    <li>Updated the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+    <li>Make it easier to find information and guidance on the Notify website</li>
+    <li>Make it easier for services to request to go live (private beta)</li>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/features/roadmap.html:62]: Should we link to https://www.notifications.service.gov.uk/pricing/trial-mode for more context?

File: app/templates/views/guidance/features/roadmap.html [hunk 4]
```diff
@@ -29,29 +29,27 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Let services create letter templates that comply with the <a class="govuk-link govuk-link--no-visited-state" href="https://law.gov.wales/culture/welsh-language/welsh-language-act-1993">Welsh Language Act 1993</a></li>
+    <li>Make it easier for users to join a live service</li>
     <li>Let services attach pages to letter templates</li>
-    <li>Explore ways to add images, QR codes and more formatting options to email templates</li>
```
Comment (karlchillmaid) [inline app/templates/views/guidance/features/roadmap.html:35]: I guess we’re not moving this to Done because it was exploratory work?

**Review by karlchillmaid — APPROVED** [review]

File: app/templates/views/guidance/features/roadmap.html (hunk 4 above)
Comment (saimaghafoor) [reply app/templates/views/guidance/features/roadmap.html:35]: I guess so, some of which we're working on currently such as adding QR codes, more formatting options like links/URLS (and subheadings, call to action) - not sure about the images though

File: app/templates/views/guidance/features/roadmap.html (hunk 3 above)
Comment (saimaghafoor) [reply app/templates/views/guidance/features/roadmap.html:62]: I'm not sure the trial mode page (or any other content) is explicit on how it's easier to request to go-live?

Comment (karlchillmaid) [reply app/templates/views/guidance/features/roadmap.html:62]: Yeah, good point.
I wasn’t so much thinking about linking to something that describes the work as providing some additional context for anyone interested.

**Pull request merged**